src/main/scala/scala/util/Using.scala

```diff
@@ -1,130 +1,241 @@
-package scala.util
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
```

Comment:
I'm not sure if this is the correct way to handle copyright/the copyright header. Would love some input from those more knowledgeable